PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/model/StringSequenceNode.java

```diff
@@ -0,0 +1,165 @@
+package nl.tudelft.pl2016gr2.model;
+
+import java.util.Collection;
+import java.util.HashSet;
+
+/**
+ * A simple implementation of <code>Node</code> that represents the DNA sequence as String.
+ *
+ * @author Wouter Smit
+ */
+public class StringSequenceNode extends AbstractNode {
+
+  private String sequence;
+  private HashSet<String> genomes;
```

Comment:
Ok I got a better estimate for the size of the original graph if the genome names are replaced by ids, I simply changed every genome addition into an addition of a string of 2 characters (2 characters is the same size as an integer), resulting in:
```
before = 2750088
after = 174954064
```
So 164MB